docs/products/postgresql/concepts/timescaledb.rst

```diff
@@ -0,0 +1,31 @@
+About TimescaleDB
+=================
+
+TimescaleDB is an open-source database designed to make your existing relational database scalable for time series data. `TimescaleDB <https://github.com/timescale/timescaledb>`_ is available as a PostgreSQL extension on Aiven.
```

Comment:
Could we move the main TimescaleDB link to the first word of this sentence?

Reply:
Done, good suggestion